ui/src/app/pages/access/access-renewal.tsx

```diff
@@ -424,6 +425,59 @@ export const RenewalCardBody = (props: {
         </React.Fragment>
       ),
     ],
+    [
+      AccessModule.CTCOMPLIANCETRAINING,
+      () => (
+        <React.Fragment>
+          <Dates />
+          <div style={textStyle}>
+            You are required to complete the refreshed ethics training courses
+            to understand the privacy safeguards and the compliance requirements
+            for using the <AoU /> Dataset.
+          </div>
+          {!isRenewalCompleteForModule(moduleStatus) && (
+            <div
+              style={{
+                ...renewalStyle.complianceTrainingExpiring,
+                ...textStyle,
+              }}
+            >
+              When you have completed the training click the refresh button or
+              reload the page.
+            </div>
+          )}
+          <TimeEstimate />
+          <FlexRow style={{ marginTop: 'auto' }}>
+            <ActionButton
+              actionButtonText='Complete Training'
+              completedButtonText='Completed'
+              moduleStatus={moduleStatus}
+              onClick={() => {
+                setTrainingRefreshButtonDisabled(false);
```

Comment:
(aside, can ignore)

just want to note here that we are using the same value for RT and CT refresh disabled.  Instinctually I would want to see separate values for the 2 tiers, BUT in this case the refresh action actually does refresh both tiers of training because we ask Moodle for both values.